resources/views/individuals/partials/progress.blade.php

```diff
@@ -31,20 +31,17 @@
             <p class="stack">
                 @can('publish', $individual)
                     <button class="secondary" name="preview" value="1">{{ __('Preview page') }}</button>
+                    <x-interpretation class="interpretation--start" name="{{ __('Publish page', [], 'en') }}"
+                        namespace="publish_page" />
```

Comment:
Sorry my fault this is the preview button, which incidentally we don't have a sign language video for. The publish interpretation does need to go below line 38 in a new `@can` block. Although things are starting to get muddled here with some many `@can` and `@cannot` blocks. I'm thinking it'd probably be easier to have just one `@can`\`@else` set where the button and interpretation in the first half and a button with the necessary aria attributes, can't publish explanation and related interpretation.